robusta-android-example/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+use robusta_jni::bridge;
+
+#[bridge]
+mod jni {
+    use android_logger::Config;
+    use jni::objects::JObject;
+    use log::info;
+    use robusta_jni::convert::{Signature, IntoJavaValue, FromJavaValue, TryIntoJavaValue, TryFromJavaValue};
+    use robusta_jni::jni::JNIEnv;
+    use robusta_jni::jni::objects::AutoLocal;
+    use robusta_jni::jni::errors::Result as JniResult;
+    use robusta_jni::jni::errors::Error as JniError;
+
+    #[derive(Signature, TryIntoJavaValue, TryFromJavaValue)]
+    #[package(android.content.Context)]
+    pub struct JContext<'env: 'borrow, 'borrow> {
```

Comment:
Here maybe we should remove this struct and use the new `#[input_type]` attribute directly on the method signature to demonstrate its usage. What do you think?

Reply:
Yes it's what I've done on my side, but I've written a simple example with what I though was the solution at first.
So you agree with my solution ? There weren't another way to proceed that I didn't know (as by using this structure) ?

Reply:
Exactly, the example is exactly how I would imagine it without this PR.

I think the PR definitely has its worth! :tada: 

I would replace this "old-style" usage with the new attribute (because that will cover 99% of the usages for Android users).

Really. I think there are only minor issues left (and documentation, but that can be added later) for the PR to be merged. Thank you again for the work and the patience!